vyperlang/vyper
# Created ERC1155 ownable example/template

### What I did
I created the ERC1155 ownable template.
This template implements ownable, name / symbol / uri to be compatible with NFT exchanges like opensea.io
Full documentation in the vyper file.

### How I did it
Used the latest Vyper 0.3.2 [edit: the master-0.3.1 to be precise] and gratefully made use of the new bytes4 and dynarray functionality, kept the ERC1155 documentation and the opensea 1155 standard handy

### How to verify it
Full brownie test is included with this PR. test coverage 100%.
Please run and verify

### Commit message
Please review, test and hopefully accept the ERC1155 ownable contract

### Description for the changelog
New example ERC1155 ownable contract + test.

### Cute Animal Picture
Happy Easter :-) 

![image](https://user-images.githubusercontent.com/33023023/163700510-80698532-8d08-4fd9-a78e-374e1c69701d.png)
